auto_submit/lib/model/pull_request_state.dart

```diff
@@ -0,0 +1,10 @@
+// Copyright 2023 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// This class holds values for the current states of the pull requests we
+/// process with the autosubmit service.
+class PullRequestState {
```

Comment:
Do we need a separate file for this purpose? Does it make sense to consolidate with `pull_request_change_type`?

Reply:
Ah yeah I can add it to that file.